runtime/ops/tls.rs

```diff
@@ -398,6 +398,12 @@ async fn op_accept_tls(
     .try_or_cancel(cancel)
     .await?;
 
+  let sni = tls_stream
```

Comment:
Wrap this in an if statement to check if unstable is enabled. If it is not, sni should be None.